cli/src/main.rs

```diff
@@ -33,6 +33,11 @@ use std::{
     path::PathBuf,
 };
 use structopt::StructOpt;
+use subxt_codegen::GeneratedTypeDerives;
+use syn::{
+    Ident,
+    __private::Span,
```

Comment:
Prefer not to import this since it is marked as `__private`. So you can either use `proc_macro2::Span` or some other way of constructing the `syn::Path`. If you still want to construct an `Ident` you should be able to use `quote::format_ident`, or you can use `syn::parse_str` or similar to go straight to a `Path`.